vnext/src/Libraries/Components/View/ReactNativeViewViewConfig.windows.js

```diff
@@ -140,7 +140,6 @@ const ReactNativeViewConfig = {
     accessibilityLabel: true,
     accessibilityLiveRegion: true,
     accessibilityRole: true,
-    accessibilityStates: true, // TODO: Can be removed after next release
```

Comment:
This and the comment were stock 0.62. We should probably leave this as is.

Reply:
This is weird - I don't see any mention of accessibilityStates in the RN docs.  Is this property not actually removed yet?

Reply:
It's removed to my understanding. You could quickly search their view managers in their 0.62 branch though. My guess it that fb has a view manager internally that still allows it, or just forgot to regenerate

Reply:
There were other places in the 0.62 upgrade where it seemed like support was truly removed.

Reply:
OK I undid this to prepare for backporting to 61

---
In reply to: [409173188](https://github.com/microsoft/react-native-windows/pull/4617#discussion_r409173188) [](ancestors = 409173188)